### ISSUE TITLE
wallet-desktop.jar shows wrong methods for contract execution

### ISSUE THREAD

**Issue by tkoen93** [CREDITSCOM/ewa#3]
While testing some stuff on the devs & dapps testnet I noticed the following in the wallet-desktop.jar

![photo_2019-06-09_09-51-10](https://user-images.githubusercontent.com/43822286/59156461-1d24e800-8a9c-11e9-94b4-d433ae212098.jpg)

The method shown by the wallet-desktop for my contract do not match the methods in the contract (https://monitor.credits.com/Devs-Dapps-Testnet/Contract/F6bsQbbCCrZdtmCMKabC2aVHzXTJacC9MpU4gYZWYPaR)

When I request methods via the `SmartContractDataGet` I do receive the correct method. So I think it must be related to the wallet-desktop.jar somehow

![image](https://user-images.githubusercontent.com/43822286/59156473-6412dd80-8a9c-11e9-8e86-3a7c92086c9c.png)

This simple contract works fine when sending a transaction to it via the API.

**Comment by IGoryunov** [comment]
@tkoen93 thanks for detailed reporting. 
Unfortunately I can't reproduce this bug. Please pull last commit from master branch, rebuild and try again.
<img width="840" alt="image" src="https://user-images.githubusercontent.com/14080974/59432031-05789700-8def-11e9-9e55-b3c34f2aafbc.png">

**Comment by tkoen93** [comment]
@IGoryunov This issue occured when I first deployed the following contract (https://monitor.credits.com/Devs-Dapps-Testnet/Contract/DxCmxu1hsmKAzWW34ANty2DVeX1AsD1n8NXY771812jF):
```java
import com.credits.scapi.annotations.*;
import com.credits.scapi.v0.*;

public class simpleContract extends SmartContract {

 public String simple;
 public int i;

 public simpleContract() {
 simple = "Simple contract return";
 i = 0;
 }
 
 public String getSimple() {
 i++;
 return String.valueOf(i) + " " + simple;
 }
}
```

After that I deployed the contract (https://monitor.credits.com/Devs-Dapps-Testnet/Contract/F6bsQbbCCrZdtmCMKabC2aVHzXTJacC9MpU4gYZWYPaR)
```java
import com.credits.scapi.annotations.*;
import com.credits.scapi.v0.*;

public class simpleContract extends SmartContract {

 public int i;

 public simpleContract() {
 i = 0;
 }
 
 public int getSimple() {
 i++;
 return i;
 }
}
```

As shown in my previous post you can see that the API returns the correct method, also probably the reason why you see the correct method upon requesting the methods for this contract.

As it looks like this only happens when you deploy contracts yourself. To reproduce you might want to try and deploy the two contracts.

**Comment by IGoryunov** [comment]
Could you check please last commit on master branch is 284b769c73e608d960ff1a9c847ab9601569513a? I sequentially reproduced actions that you describe above, but didn't get similar result

<img width="1286" alt="image" src="https://user-images.githubusercontent.com/14080974/59440882-5644bb80-8e00-11e9-9c25-47634cd91294.png">

**Comment by tkoen93** [comment]
Seems to work allright in the wallet-desktop.jar that's included in the 4.2.417 testnet version.
